logstreams/src/main/java/io/zeebe/logstreams/rocksdb/ZbRocksDb.java

```diff
@@ -229,6 +228,8 @@ public boolean exists(
       ColumnFamilyHandle columnFamily, DirectBuffer key, MutableDirectBuffer value) {
     EnsureUtil.ensureArrayBacked(key, value);
 
+    // todo(npepinpe): afaik keyMayExist doesn't necessarily fill the StringBuilder completely, so
+    // it doesn't seem particularly useful
```

Comment:
could we use null?

Reply:
Unfortunately not, it throws an NPE :(

I thought about using a static one, but I'm not sure what's more performant: allocating a dummy one and throwing it away immediately (so gen 0 object), or having a long lived object which we clear.

Reply:
Maybe in the future is some more time to write some performance tests for our states, then we can check that.